--- origbeef/ui/index.php	2007-04-03 22:57:48.000000000 -0400
+++ beef/ui/index.php	2009-08-18 23:57:12.000000000 -0400
@@ -48,6 +48,7 @@
 	<script src="../js/autorun.js" type="text/javascript"></script>
 	<script src="../js/zombie.js" type="text/javascript"></script>
 	<script src="../js/module.js" type="text/javascript"></script>
+	<script src="../js/msf.js" type="text/javascript"></script>
 
 <?php
 	// config check
